augraphy/augmentations/geometric.py

```diff
@@ -24,7 +24,13 @@ class Geometric(Augmentation):
            the image rotation.
     :type rotate_range: tuple, optional
     :param randomize: Flag to apply random geometric transformations.
-    :param randomize: int, optional
+    :type randomize: int, optional
+    :param padding: Padding amount on each [left, right, top, bottom] side.
+    :type padding: tuple, optional
+    :param padding_type: Padding methods, select from fill,duplicate and mirror.
```

Comment:
So there will be 3 padding methods, `fill` which is fill with any value, `duplicate` is to duplicate the image, including their orientation, and `mirror` which is duplicate but in inversed orientation.

For example:

Fill red value on the left:
![image](https://user-images.githubusercontent.com/52521979/225485057-2f0c818e-eb1a-423f-b001-43520046d1ea.png)

Duplicate on the left:
![image](https://user-images.githubusercontent.com/52521979/225484969-ee9d0909-9bf8-417f-8fcf-f2c14d632c24.png)

Mirror on the left:
![image](https://user-images.githubusercontent.com/52521979/225485014-f4ab9883-7cfd-438a-9904-4768174bb392.png)